petab/petablint.py

```diff
@@ -82,9 +86,15 @@ def parse_cli_args():
                 folder=args.directory,
             )
 
-    if not args.model_name and not args.sbml_file_name \
-        and not args.condition_file_name and not args.measurement_file_name \
-            and not args.parameter_file_name:
+    if (args.yaml_file_name
+            and (args.sbml_file_name or args.condition_file_name
```

Comment:
and any([...]) might be nicer here (no opinion though)

Reply:
Will consider it next time ;)